tapfreighter/chain_porter.go

```diff
@@ -311,8 +314,6 @@ func (p *ChainPorter) QueryParcels(ctx context.Context,
 // requests, and attempt to complete a transfer. A response is sent back to the
 // caller if a transfer can be completed. Otherwise, an error is returned.
 func (p *ChainPorter) mainEventLoop() {
-	defer p.Wg.Done()
```

Comment:
We should add a `NOTE` comment to this method that it MUST be run as a goroutine.
Normally, the `defer p.Wg.Done()` at the start of a method indicates this to someone reading the code.